alacritty/src/renderer/mod.rs

```diff
@@ -1513,14 +1520,17 @@ impl Atlas {
             gl::PixelStorei(gl::UNPACK_ALIGNMENT, 1);
             gl::GenTextures(1, &mut id);
             gl::BindTexture(gl::TEXTURE_2D, id);
+            // We use an RGBA atlas that can handle colored glyphs.
+            // As they usually are outnumbered by regular text glyphs, this is slightly wasteful.
+            // Another way would be to have a separate atlas for RGBA and render in two passes.
```

Comment:
Could add add explanation there, that one more Altlas approach doesn't give us much benefits over this and link to that PR?